Cargo.toml

```diff
@@ -17,12 +17,12 @@ edition = "2018"
 all-features = true
 
 [dependencies]
-async-compression = { version = "0.3.1", features = ["brotli", "deflate", "gzip", "stream"], optional = true }
-bytes = "0.5"
+async-compression = { git = "https://github.com/aknuds1/async-compression", rev = "e4d903b8ff9972f056f714c61bd9d0f3321a4463", features = ["brotli", "deflate", "gzip", "stream"], optional = true }
```

Comment:
Do you really need the `rev`, or is it just pointing to master?

Reply:
It's more the other way around, master is pointing to this revision. At least it was, which is why I prefer pinning the revision, since a branch by its nature will change and you have no guarantee it will keep working.

The intent is also to switch to official versions of dependencies, before finalizing this PR. The current dependencies on Git commits just represent what I could make work.